renaissance-harness/src/main/scala/org/renaissance/harness/RenaissanceSuite.scala

```diff
@@ -145,6 +148,35 @@ object RenaissanceSuite {
     }
   }
 
+  private def getVmStartNanos = {
+    //
+    // Get nanoTime() reading from between two
+    // distinct currentTimeMillis() readings.
+    //
+    val initMillis = System.currentTimeMillis()
+    while (System.currentTimeMillis() == initMillis) {
+      // Wait for the next reading.
+    }
+
+    val currentNanosBefore = System.nanoTime()
```

Comment:
Don't you want to make it so that `currentNanosBefore` is the last timestamp taken before millis change, not the first one ?